.CI/trigger_travis_conda-forge.github.io.sh

```diff
@@ -0,0 +1,39 @@
+"""
+Trigger the conda-forge.github.io Travis job to restart.
+
+"""
+import requests
+import six
```

Comment:
I love how they give that emoticon eyes. No worries. I'll try to type really fast this time. :wink: